src/main/java/com/datatorrent/netlet/DefaultEventLoop.java

```diff
@@ -513,14 +512,18 @@ public String toString()
 
             });
           }
-        }
-        catch (IOException ie) {
-          l.handleException(ie, DefaultEventLoop.this);
+        } catch (UnresolvedAddressException e) {
+          logger.error("Inet Address {} is not resolvable.", address, e);
```

Comment:
What about logging it as trace level and using DTLoggerFactory to enable it for com.datatorrent.netlet?

Also earlier PR did register the UncaughtExceptionHandler which logs it using error. So even if you do not log it here - you will see it in logs, right? I like that centralized logging as it open up an avenue to have the clients register their unhandled exception handler and have them do whatever they want in their.

-
Chetan